tests/host_vars/openstack1.yml

```diff
@@ -28,4 +28,4 @@ neutron_local_ip: "{{ ansible_host }}"
 
 swift_replication_address: 10.4.1.101
 
-swift_storage_address: "{{ storage_address }}"
+swift_storage_address: "{{ storage_address }}"
```

Comment:
at some point you should change your text editor to not have these :p 

POSIX!